### PR TITLE
Docs: Integrating the VTEX login

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

This PR:
- Adds the new doc _Integrating the VTEX Login_, part of the go-live track.
- Moves the _Setting up a VTEX account for FastStore_ doc to the go-live track.

🔗 [Preview](https://faststore-site-git-docs-integrating-vtex-login-faststore.vercel.app/docs/go-live/3-integrating-the-the-vtex-login)

### CODE REVIEW

File: apps/site/pages/docs/go-live/overview.mdx [hunk 1]
```diff
@@ -44,8 +44,8 @@ To ensure that you have set up all configurations for your store to run smoothly
 <details>
 <summary>Ensure that your account edition is set to `vtex.edition-store@5.x` Edition app in your master workspace.</summary>
 
-| Management              | Reference                                                                                                                                                                       |
-| :---------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Management              | Reference                                                                                                                                                                    |
+| :---------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Agency and VTEX Support |  If your account uses a different Edition App version, [open a support ticket](https://help.vtex.com/en/support) requesting the installation of the vtex.edition-store@5.x.  |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/overview.mdx:49]: > If your account uses a different Edition App version, [open a support ticket](https://help.vtex.com/en/support) requesting the installation of the vtex.edition-store@5.x.
Questions:
1. Does the recommendation also apply to Legacy stores? 
2. Should they have the editor set in `master`?

File: apps/site/pages/docs/go-live/3-integrating-the-the-vtex-login.mdx [hunk 2]
```diff
@@ -13,4 +13,45 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
...
+
+The VTEX Login is provided by the `vtexcommercestable` environment and uses a subdomain. Therefore, to successfully integrate the VTEX Login with your FastStore project, you must add a redirect back to the FastStore environment and set up the necessary auth cookies to authenticate client requests and maintain session information. This way, once shoppers finish logging in to your store, they can be redirected back to your website's main domain.
+
+---
+
+## Before you start
+
+- Ensure you have already [configured the custom domain name of your FastStore website](/docs/go-live/2-configuring-external-dns).
+- Stores that currently use Portal (Legacy) technology are required to update their `vtex.edition-business@0.x` to `vtex.edition-store@5.x,`  and to migrate their secure domain to VTEX IO. To make these changes, please open a support ticket with the [VTEX support team](https://help.vtex.com/support) and request that the [Edition App](https://developers.vtex.com/docs/guides/vtex-io-documentation-edition-app) installed in your account be updated and that your secure domain be migrated from Portal (Legacy) to VTEX IO.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/3-integrating-the-the-vtex-login.mdx:27]: Is there any other information to highlight for Legacy stores migrating to FastStore?

File: apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx [hunk 3]
```diff
@@ -13,4 +13,47 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
...
+
+2. Install the FastStore plugin for the VTEX IO CLI:
+
+```
+vtex plugins install faststore
+```
+
+3. Set up your VTEX account for FastStore by running the following command:
+```
+vtex faststore setup
```
Comment (emersonlaurentino) [inline apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx:56]: We intend to discontinue this to prioritize the onboarding to be done through the faststore app directly from the admin. cc @gvc

Comment (Mariana-Caetano) [reply apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx:56]: @emersonlaurentino is that already a reality?

File: apps/site/pages/docs/go-live/3-integrating-the-the-vtex-login.mdx (hunk 2 above)
Comment (gvc) [reply apps/site/pages/docs/go-live/3-integrating-the-the-vtex-login.mdx:27]: @Mariana-Caetano I'm trying to get more info from other stores that have done this migration and will send it your way about what they've done.

File: apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx (hunk 3 above)
Comment (gvc) [reply apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx:56]: I'm not sure if this is a reality yet, but I agree that we should move in that direction. I'd leave the step in the readme right now and remove it once we're certain that this is a reality.

File: apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx [hunk 4]
```diff
@@ -13,4 +13,48 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
...
+vtex login {account}
+```
+
+2. Install the FastStore plugin for the VTEX IO CLI:
+
+```
+vtex plugins install faststore
+```
+
+{/*3. Set up your VTEX account for FastStore by running the following command:
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx:54]:
```suggestion
3. Set up your VTEX account for FastStore by running the following command:
```

File: apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx [hunk 5]
```diff
@@ -13,4 +13,48 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
...
+```
+vtex faststore setup
+```
+/*}
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx:58]:
```suggestion
```

File: apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx [hunk 6]
```diff
@@ -13,4 +13,47 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
...
+2. Install the FastStore plugin for the VTEX IO CLI:
+
+```
+vtex plugins install faststore
+```
+
+3. Set up your VTEX account for FastStore by running the following command:
+```
+vtex faststore setup
+```
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx:57]:
```suggestion
```

File: apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx [hunk 7]
```diff
@@ -13,4 +13,47 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
...
+
+2. Install the FastStore plugin for the VTEX IO CLI:
+
+```
+vtex plugins install faststore
+```
+
+3. Set up your VTEX account for FastStore by running the following command:
+```
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/1-setting-up-vtex-account-for-faststore.mdx:55]:
```suggestion
```

File: apps/site/pages/docs/go-live/3-integrating-the-the-vtex-login.mdx (hunk 2 above)
Comment (Mariana-Caetano) [reply apps/site/pages/docs/go-live/3-integrating-the-the-vtex-login.mdx:27]: @gvc any news? 👀

Comment (Mariana-Caetano) [reply apps/site/pages/docs/go-live/3-integrating-the-the-vtex-login.mdx]: @gvc I've made some changes based on the doc you shared about the legacy store. The changes you can check here: https://github.com/vtex/faststore/pull/1807/files#diff-dc9b1b7354ff76188e7358360b4f3b6d8db9871931fd7ce6dd3473ee396ab265R23-R33

Do you think we can move forward and merge this PR?

**Review by gvc — APPROVED** [review]

**Pull request merged**